src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -38,6 +38,19 @@ public class CommandLineConfiguration
 
             foreach (var symbol in symbols)
             {
+                foreach (var childSymbol in ((ISymbolSet)symbol.Children).FlattenBreadthFirst(o => o.Children))
```

Comment:
Why the `ISymbolSet` cast here? It should not be needed.

```suggestion
                foreach (var childSymbol in symbol.Children.FlattenBreadthFirst<Symbol>(o => o.Children))
```